src/modules/fundingManager/oracle/FM_PC_ExternalPrice_Redeeming_v1.sol

```diff
@@ -0,0 +1,769 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
There are no tests covering this contract, and we believe we have found some errors that would definitely have been caught with a test.